src/Features/CSharp/Portable/Completion/CompletionProviders/PropertySubPatternCompletionProvider.cs

```diff
@@ -120,12 +177,35 @@ private static SyntaxToken TryGetOpenBraceOrCommaInPropertyPatternClause(SyntaxT
             var token = tree.FindTokenOnLeftOfPosition(position, cancellationToken);
             token = token.GetPreviousTokenIfTouchingWord(position);
 
-            if (!token.IsKind(SyntaxKind.CommaToken, SyntaxKind.OpenBraceToken))
+            if (token.IsKind(SyntaxKind.CommaToken, SyntaxKind.OpenBraceToken))
             {
-                return default;
+                return token.Parent.IsKind(SyntaxKind.PropertyPatternClause) ? makeResult(token) : default;
```

Comment:
this is a bit weird.  yo uahve the local functino.  but it's called in one place.